config/hancestro.yml

```diff
@@ -4,16 +4,17 @@ idspace: HANCESTRO
 base_url: /obo/hancestro
 
 products:
-- hancestro.owl: https://raw.githubusercontent.com/EBISPOT/ancestro/master/hancestro_inferred.owl
+- hancestro.owl: https://raw.githubusercontent.com/EBISPOT/hancestro/main/hancestro.owl
 
 term_browser: ontobee
 example_terms:
 - HANCESTRO_0004
 
 entries:
 - exact: /hancestro.owl
-  replacement: https://raw.githubusercontent.com/EBISPOT/ancestro/master/hancestro.owl
-- exact: /hancestro_inferred.owl
```

Comment:
It is against the spirit of OBO PURLs to simply remove them. Can you instead redirect this to https://raw.githubusercontent.com/EBISPOT/hancestro/main/hancestro.owl?

Reply:
I could absolutely redirect but there's no direct equivalent file, so the resulting file would look different. If that's not a problem, I'll put it back in

Reply:
@matentzn the new `hancestro.owl` will be updated with every release. The old `hancestro_inferred.owl` will never be updated again. There is a legacy version of release 2.8 at `https://github.com/EBISPOT/hancestro/blob/main/retired_artefacts/hancestro_inferred.owl` but this is now obviously out of sync with the ontology content and will diverge more as we add new classes. I'm not sure which is better - pointing the old PURL to a new artefact that looks different or pointing it to the static last version of the old artefact that will diverge more with every new release.

Reply:
I personally think its better to link to the retired artefact, because it makes it at least explicit that the file is no longer maintained. But we are not prescriptive here. How do you want to proceed?

Reply:
Give me 5 minutes to push an updated version of the config pointing to the old file; I think I prefer that approach. Then the PR is ready to merge.